packages/teams-js/test/public/chat.spec.ts

```diff
@@ -83,16 +84,34 @@ describe('chat', () => {
             message: 'someMessage',
           };
 
+          const normalizedChatRequest = {
+            members: ['someUPN'],
```

Comment:
>members: ['someUPN'], [](http://example.com/codeflow?start=12&length=21)

Note that this test now validates that the request contains the single user in an array